From 16722fa78377b3bc9a37f228658454437a88494f Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 5 Dec 2016 23:26:32 +0800
Subject: [PATCH 1308/5242] MLK-13567 ARM: dts: imx7ulp: disable iomuxc by
 default

commit  ae10ff3bcf276774e0deebcc17403fb274ca18f9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable i.MX7ULP's IOMUXC now since there is no module
using it and after kernel boot up, below failed message
will come out:

imx7ulp-pinctrl 4103d000.iomuxc: fail to probe dt properties
imx7ulp-pinctrl: probe of 4103d000.iomuxc failed with error -22

Any module who needs it can enable it anytime.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index f432ec6..284a5c6 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -460,6 +460,7 @@
 			compatible = "fsl,imx7ulp-iomuxc-0";
 			reg = <0x4103D000 0x1000>;
 			fsl,mux_mask = <0xf00>;
+			status = "disabled";
 		};
 
 		iomuxc1: iomuxc1@40ac0000 {
-- 
1.7.9.5

